### PR TITLE
Add SizeForm tests and fix eslintignore filename

### PR DESCRIPTION
**Pull request by TanyaTop**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Screenshots
photo_2021-07-23 11 21 30

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/components/forms/size-form/tests/size-form.test.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      .find('#modelId')
+      .props()
+      .onChange({
+        target: {
+          value: selectValue,
+          name: labels.en.modelName
+        }
+      });
+
+    expect(mockSetFieldValue).toHaveBeenCalledTimes(1);
```
Comment (sergklukvin) [inline src/components/forms/size-form/tests/size-form.test.js:89]: expect should use on the end

File: src/components/forms/size-form/tests/size-form.test.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  it('Should have default props', () => {
+    expect(SizeForm.defaultProps).toBeDefined();
+    expect(SizeForm.defaultProps.id).toBe(sizeDefaultProps.id);
+    expect(SizeForm.defaultProps.size).toEqual(sizeDefaultProps.size);
+  });
+
+  it('Should have props', () => {
+    wrapper = mount(<SizeForm />);
+    expect(wrapper.props().id).toBe(sizeDefaultProps.id);
+    expect(wrapper.props().size).toEqual(sizeDefaultProps.size);
```
Comment (sergklukvin) [inline src/components/forms/size-form/tests/size-form.test.js:124]: expect should use on the end

**Review by sergklukvin — DISMISSED** [review]

File: src/components/forms/size-form/tests/size-form.test.js (hunk 1 above)
Comment (TanyaTop) [reply src/components/forms/size-form/tests/size-form.test.js:89]: What do you mean? 
It is a best practice to use **expect**

File: src/components/forms/size-form/tests/size-form.test.js (hunk 2 above)
Comment (TanyaTop) [reply src/components/forms/size-form/tests/size-form.test.js:124]: The same

**Review by halaburdataras — APPROVED** [review]

**Review by BohdanShchur — APPROVED** [review]

**Pull request merged**